weather_record.py

```diff
@@ -3,12 +3,9 @@ class WeatherRecord:
     def __init__(self, record_item) -> None:
 
         self.date = record_item['PKT']
-        self.min_temp = record_item['Mean TemperatureC']
+        self.min_temp = record_item['Min TemperatureC']
         self.max_temp = record_item['Max TemperatureC']
-        self.avg_temp = record_item['Min Temperature']
+        self.mean_temp = record_item['Mean TemperatureC']
         self.min_humidity = record_item[' Min Humidity']
         self.max_humidity = record_item['Max Humidity']
```

Comment:
try to keep variable and keys naming convention same. and you can also use pandas for column operations

Reply:
Okay sure. the confusion was that the task document use the word "average" and data files has the word 'mean'. So, in this case which one i have to use?

Reply:
it might be `mean temperature` of a day but not sure